static/app/types/group.tsx

```diff
@@ -352,6 +352,10 @@ export interface GroupActivitySetIgnored extends GroupActivityBase {
     ignoreCount?: number;
     ignoreDuration?: number;
     ignoreUntil?: string;
+    /**
+     * Archived until escalating
+     */
```

Comment:
A multi-line comment feels like overkill, IMHO, but this is an utter nit. Feel free to take or leave. 

```suggestion
    // Archived until escalating
```